docs/assets/scss/schema-docs.scss

```diff
@@ -81,68 +89,99 @@
 
   span.formal-name {
     @include supporting-text();
+
     a {
       @include u-text('bold');
     }
   }
 }
 
 .xml-index {
-  @include doc-index('green-cool-20','green-cool-80');
+  @include doc-index('green-cool-20', 'green-cool-80');
 }
 
 .json-index {
-  @include doc-index('blue-20','blue-80');
+  @include doc-index('blue-20', 'blue-80');
 }
 
-.xml-outline, .json-outline {
+.xml-outline,
+.json-outline {
   @include model-text();
   hyphens: none;
 
   div.OM-entry p {
     line-height: 1.5;
   }
+
   details.OM-entry {
     .model-container {
       margin-left: 2.5em;
     }
+
     &[open] {
       summary {
         .show-closed {
           display: none;
         }
       }
     }
+
     &:not([open]) {
       summary {
         .show-closed {
           display: initial;
         }
       }
     }
+
     summary {
       cursor: pointer;
       list-style-position: outside;
       line-height: 1.5;
     }
   }
+
   /*span.OM-cardinality {  }*/
-  .model-container + p {
+  .model-container+p {
     margin-top: 0;
   }
 
-  a, a:visited {
+  a,
+  a:visited {
     text-decoration: none;
   }
 
-  div.OM-choice {
+  div.OM-choices {
+    @include u-margin-left(-2);
     @include u-padding-left(2);
+    background-color: lightgray;
+    width: max-content;
+
+    div.OM-choice {
+      @include u-padding-left(0);
+    }
+
+    // Nested choices alternate color for readability
+    div.OM-choices {
+      background-color: white;
+
+      div.OM-choices {
+        background-color: lightgray;
+
+        // Add nested colors as needed
+      }
+    }
   }
```

Comment:
How about this? A blue closer to our current palette and more aggressive indentation?

![image](https://user-images.githubusercontent.com/94922603/222843425-cd52707b-6d09-4285-9d86-5a39ad44a8b5.png)

![image](https://user-images.githubusercontent.com/94922603/222843501-430cd94d-2669-48f1-ae7e-06ada7a3a4ba.png)


```suggestion
  div.OM-choices {
    @include u-margin-left(-2);
    @include u-padding-left(2);
    background-color: #d9e8f6;
    width: max-content;

    div.OM-choice {
      @include u-padding-left(4);
    }

    // Nested choices alternate color for readability
    div.OM-choices {
      background-color: white;

      div.OM-choices {
        background-color: lightgray;
        // Add nested colors as needed
      }
    }
  }
```